kolibri/plugins/coach/assets/src/routes/planExamRoutes.js

```diff
@@ -37,20 +37,26 @@ export default [
     name: PageNames.EXAM_CREATION_ROOT,
     path: '/:classId/plan/quizzes/new',
     component: CreateExamPage,
-    children: [
-      {
-        name: PageNames.QUIZ_SECTION_EDITOR,
-        path: ':section_id/edit',
-      },
-      {
-        name: PageNames.QUIZ_REPLACE_QUESTIONS,
-        path: ':section_id/replace-questions',
-      },
-      {
-        name: PageNames.QUIZ_SELECT_RESOURCES,
-        path: ':section_id/select-resources',
-      },
-    ],
+  },
+  {
+    name: PageNames.QUIZ_SECTION_EDITOR,
+    path: '/:classId/plan/quizzes/new/:section_id/edit',
+    component: CreateExamPage,
+  },
+  {
+    name: PageNames.QUIZ_REPLACE_QUESTIONS,
+    path: '/:classId/plan/quizzes/new/:section_id/replace-questions',
+    component: CreateExamPage,
+  },
+  {
+    name: PageNames.QUIZ_SELECT_RESOURCES,
+    path: '/:classId/plan/quizzes/new/:section_id/select-resources/:topic_id?',
+    component: CreateExamPage,
+  },
+  {
+    name: PageNames.BOOK_MARKED_RESOURCES,
+    path: '/:classId/plan/quizzes/new/:section_id/select-resources/bookmarks/:topic_id?',
```

Comment:
I was curious, what does topic_id mean in this context? We can't bookmark topics, might this be more semantically named?